native/package.json

```diff
@@ -64,7 +64,7 @@
     "react-native-pdf": "6.2.0",
     "react-native-permissions": "2.0.9",
     "react-native-progress": "^4.0.3",
-    "react-native-render-html": "^4.2.0",
+    "react-native-render-html": "^5.0.0-alpha.1",
```

Comment:
5.0.0-alpha.2 is out :) probably not worth upgrading though

Reply:
will change to 5.0.0-alpha.2. See [here](https://github.com/Integreat/integreat-app/pull/48/files#r515019513)